### ISSUE TITLE
Label control does not support UIA accessiblility

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#3206]
* .NET Core Version: 
.NET Core 5.0

 Version:   5.0.100-alpha.1.20073.10
 Commit:    29f4d693a9

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.18363
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.100-alpha1-05536\

Host (useful for support):
  Version: 5.0.0-alpha.1.20072.3
  Commit:  c3dc1fdfdc

* Have you experienced this same bug with .NET Framework?: 
Yes

**Problem description:**

![image](https://user-images.githubusercontent.com/23213980/80997736-716e4e80-8e4a-11ea-8c75-6ebf11bffc97.png)

Accessibility providers:
"[pid:31748,providerId:0x11CFA Main:Nested [pid:31900,providerId:0x11CFA Annotation(parent link):Microsoft: Annotation Proxy (unmanaged:uiautomationcore.dll);
Main:Microsoft: MSAA Proxy (unmanaged:uiautomationcore.dll)];
Hwnd(parent link):Microsoft: HWND Proxy (unmanaged:uiautomationcore.dll)]"

**Expected behavior:**
There should be UIA accessibility provider from WinForms

**Minimal repro:**
Create WinForms app with Label control and analyze it using Inspect.

**Related item:**
This work item is the sub of #2164
